rcl_interfaces/msg/ParameterValue.msg

```diff
@@ -6,8 +6,8 @@
 uint8 type
 
 # "Variant" style storage of the parameter value.
-bool bool_value
-int64 integer_value
-float64 double_value
-string string_value
+bool bool_value False
```

Comment:
It is not supposed to work given that our specification defines `true`, `false`, `1` and `0` as the only allowed valued for `bool`. It happens to work because we account for user error in our parser by [lowercasing](https://github.com/ros2/rosidl/blob/af7e2700ab30a28b6d9867508ae74887fa45233d/rosidl_parser/rosidl_parser/__init__.py#L496-L502) the value. But it is an implementation detail and would not work for any other parser written following the spec so I think we should not use it in our messages